src/ripple/app/misc/ValidatorList.h

```diff
@@ -416,8 +428,8 @@ ValidatorList::onConsensusStart (
 
     auto size = rankedKeys.size();
 
-    // Do not require 80% quorum for less than 10 trusted validators
-    if (rankedKeys.size() >= 10)
+    // Require 80% quorum if there are lots of validators.
+    if (rankedKeys.size() > BYZANTINE_THRESHOLD)
     {
         // Use all eligible keys if there is only one trusted list
         if (publisherLists_.size() == 1)
```

Comment:
This could be changed to
`if (publisherLists_.size() == 1 || keyListings_.size() < MINIMUM_RESIZEABLE_UNL)`
Otherwise, we would reduce the UNL size in the else below if we were to ever make `MINIMUM_RESIZEABLE_UNL` larger than `BYZANTINE_THRESHOLD`.